GUI/src/cz/fidentis/gui/comparison_batch/BatchRegistrationConfiguration.java

```diff
@@ -1391,6 +1419,17 @@ public void setConfiguration() {
        /* if(c.getModels().size() > 0)
             facesComboBox.setSelectedIndex(c.getTemplateIndex() + 3);
        */
+       
+       //to check whether FPs can be exported once they are added, removed
+        ObservableMaster o = new ObservableMaster();
+        ExportFPButtonObserver export = new ExportFPButtonObserver(exportFpButton, 
+        GUIController.getSelectedProjectTopComponent().getViewerPanel_Batch().getListener(),
+        null);
+        RegisterFPButtonObserver register = new RegisterFPButtonObserver(registerButton, c.getFacialPoints());
+        o.addObserver(export);
+        o.addObserver(register);
+        
+        GUIController.getSelectedProjectTopComponent().getViewerPanel_Batch().setFpExportEnable(o);
```

Comment:
Mas pravdu, bude to treba opravit, zabudla som na to, ze tato metoda sa vola aj v momente ked sa prepinsa z jedneho okna do druheho